### DISCUSSION TITLE
[1.2.9] Updated Steamless

### DISCUSSION THREAD

**Discussion by BigBoiCJ** [BigBoiCJ/SteamAutoCracker#12] [Announcements]
- Updated Steamless to [v3.1.0.3](https://github.com/atom0s/Steamless/releases/tag/v3.1.0.3)
- Errors are now handled sooner (module errors are now handled)

NOTE: Deleting the config file isn't required IF you're updating from v1.2.8.

NOTE 2: Due to some env changes on my end, this release could have bugs and fail to run. You should be able to notice that the file size is considerably smaller. Please report your issues if you experience some. In the meantime, you can try running v1.2.8.

SteamAutoCracker requires a 64 bits Windows computer.

<hr /><em>This discussion was created from the release <a href='https://github.com/BigBoiCJ/SteamAutoCracker/releases/tag/1.2.9'>[1.2.9] Updated Steamless</a>.</em>

**Comment by TeamVista8** [comment]


**Reply by BigBoiCJ** [reply]
You can move the game's folder to somewhere else and Steam won't update it.

Most game updates won't update the SteamAPI files, and thus, the crack will survive as long as you don't verify the integrity of the game's files. If the SteamAPI files are updated or the game's file integrity is checked, you'll have to crack the game again using SAC.

Uninstalling Steam is not needed, and I advise you don't, as Steam is love, Steam is life.

**Comment by TeamVista8** [comment]


**Reply by BigBoiCJ** [reply]
I'm not really sure I understood what you said, but as far as I know, Half-life 2 and source games in general can't be cracked by SAC as those games use a more in-depth anti-piracy system. It can still be countered quite easily, but not by SAC.

As for the files appearing, I have no idea to be honest. SAC doesn't do any of that.

Why would you want to crack solo games that you own anyways ? If you want to keep your games for real & all or share them with someone, I can understand, and in that case you should learn about Goldberg's Steam Emu, it should be able to crack Source games, but it requires some time to understand and configure properly. Learn more on CS.RIN.RU

**Comment by TeamVista8** [comment]


**Reply by BigBoiCJ** [reply]
Yup, games won't get updated if they're not in the `common` folder. Just put them outside of the Steam folder and done.

You might want to edit your message though as replying to emails seems a bit broken and left a lot of infos in your message
